lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -435,7 +435,7 @@ function dosomething_rogue_transform_rogue_status_to_phoenix_status($status) {
     return 'pending';
   }
   else {
-    return 'rejected';
+    return 'excluded';
```

Comment:
Greatttt catch, this was definitely an error! I think it should be `flagged` instead of `excluded` though. The only things that are `rejected` in Rogue (that were reviewed in Phoenix originally) are things that were `flagged`. Based on [this function](https://github.com/DoSomething/phoenix/blob/fdaaee7c2306425e653c8c5708e7903b35d0720c/lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module#L413-L423)